File: src/main/java/com/example/resthospital/Patient/Patient.java
package com.example.resthospital.Patient;

import com.fasterxml.jackson.annotation.JsonProperty;

import java.sql.Date;

public class Patient
{

  @JsonProperty("cpr_patient")
  int cpr_patient;

  @JsonProperty("patient_name")
  String patient_name;

  @JsonProperty("blood_type")
  String blood_type;

  @JsonProperty("house_number")
  int house_number;

  @JsonProperty("street")
  String street;

  @JsonProperty("postal_code")
  int postal_code;

  @JsonProperty("city")
  String city;

  @JsonProperty("phone")
  int phone;

  @JsonProperty("gender")
  String gender;

  @JsonProperty("last_update") Date last_update;

  @JsonProperty("patient_status")
  String patient_status;

  @JsonProperty("doctorId")
  int doctorId;

  @JsonProperty("roomID")
  int roomID;

}

File: Models/Patient.cs
using System;
using System.Collections;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using HotChocolate;
using Microsoft.EntityFrameworkCore;

namespace hospitalGQL.Models
{
    
    public class Patient
    {
        [Key] 
        public long cpr_patient { get; set; }
        
        public string patient_name { get; set; }
        public string blood_type { get; set; }
        public long house_number { get; set; }
        public string street { get; set; }
        public long postal_code { get; set; }
        public string city { get; set; }
        public long phone { get; set; }
        public string gender { get; set; }
        public DateTime  last_update { get; set; }
        public string patient_status { get; set; }

        public long cpr_doctor { get; set; }

       public long DoctorId { get; set; }
       
             
       public long RoomID { get; set; }
        public Doctor Doctor { get; set; }
        
  
        public Room Room { get; set; }
        
        public ICollection<Prescription> Prescriptions { get; set; } = new List<Prescription>();
        
    }
}